udm-le.sh

```diff
@@ -38,6 +38,13 @@ for DOMAIN in $(echo $CERT_HOSTS | tr "," "\n"); do
 	HOSTS_ARGS="${HOSTS_ARGS} -d ${DOMAIN}"
 done
 
+# Check for aws directory, add that mount point to the lego container
+# TODO add support for AWS_PROFILE
```

Comment:
AWS_PROFILE would have to be passed as an environment variable through udm-le.env to be able to have acme use a particular profile's keys/config.   Maybe it shouldn't even be a ToDo since this work is to avoid having to use environment variables.  I'll take out that line.